.stylelintrc.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": ["stylelint-config-standard-scss"],
+  "extends": ["@edx/stylelint-config-edx"],
   "rules": {
```

Comment:
hmm, I think we will also need to remove most of our rules, otherwise there is no point in using config from edX, because this way we just override a lot of stuff

Reply:
Thanks, checked all the config rules. Removed one rule, we need the rest)